src/main/java/teammates/ui/controller/Action.java

```diff
@@ -216,6 +214,7 @@ private boolean doesUserNeedToLogin(UserType currentUser) {
         return false;
     }
 
+    @SuppressWarnings("PMD.EmptyIfStmt")
```

Comment:
Ok, this should be fine then. SuppressWarnings should be preferable to NOPMD here since it specifically targets this rule I guess, and we'll notice if someone eventually tries to add an empty if statement anyway.